api/src/test/java/bio/terra/tanagra/app/controller/UnderlaysApiControllerTest.java

```diff
@@ -24,13 +24,36 @@ public class UnderlaysApiControllerTest extends BaseSpringUnitTest {
+          "{\"key\":\"concept_code\",\"width\":120,\"title\":\"Code\"}" +
+          "],";
+
+  private static String criteriaConfigs = "[" +
+          "{\"type\":\"concept\",\"title\":\"Conditions\",\"defaultName\":\"Contains Conditions Codes\",\"plugin\":{\"columns\":" + columns + "\"entities\":[{\"name\":\"condition\",\"selectable\":true,\"hierarchical\":true}]}}," +
+          "{\"type\":\"concept\",\"title\":\"Procedures\",\"defaultName\":\"Contains Procedures Codes\",\"plugin\":{\"columns\":" + columns + "\"entities\":[{\"name\":\"procedure\",\"selectable\":true,\"hierarchical\":true}]}}," +
+          "{\"type\":\"concept\",\"title\":\"Observations\",\"defaultName\":\"Contains Observations Codes\",\"plugin\":{\"columns\":" + columns + "\"entities\":[{\"name\":\"observation\",\"selectable\":true}]}}," +
+          "{\"type\":\"concept\",\"title\":\"Drugs\",\"defaultName\":\"Contains Drugs Codes\",\"plugin\":{\"columns\":" + columns + "\"entities\":[{\"name\":\"ingredient\",\"selectable\":true,\"hierarchical\":true},{\"name\":\"brand\",\"sourceConcepts\":true,\"attributes\":[\"concept_name\",\"concept_id\",\"standard_concept\",\"concept_code\"],\"listChildren\":{\"entity\":\"ingredient\",\"idPath\":\"relationshipFilter.filter.binaryFilter.attributeValue\",\"filter\":{\"relationshipFilter\":{\"outerVariable\":\"ingredient\",\"newVariable\":\"brand\",\"newEntity\":\"brand\",\"filter\":{\"binaryFilter\":{\"attributeVariable\":{\"variable\":\"brand\",\"name\":\"concept_id\"},\"operator\":\"EQUALS\",\"attributeValue\":{\"int64Val\":0}}}}}}}]}}," +
+          "{\"type\":\"attribute\",\"title\":\"Ethnicity\",\"defaultName\":\"Contains Ethnicity Codes\",\"plugin\":{\"attribute\":\"ethnicity_concept_id\"}}," +
+          "{\"type\":\"attribute\",\"title\":\"Gender Identity\",\"defaultName\":\"Contains Gender Identity Codes\",\"plugin\":{\"attribute\":\"gender_concept_id\"}}," +
+          "{\"type\":\"attribute\",\"title\":\"Race\",\"defaultName\":\"Contains Race Codes\",\"plugin\":{\"attribute\":\"race_concept_id\"}}," +
+          "{\"type\":\"attribute\",\"title\":\"Sex Assigned at Birth\",\"defaultName\":\"Contains Sex Assigned at Birth Codes\",\"plugin\":{\"attribute\":\"sex_at_birth_concept_id\"}},{\"type\":\"attribute\",\"title\":\"Year at Birth\",\"defaultName\":\"Contains Year at Birth Values\",\"plugin\":{\"attribute\":\"year_of_birth\"}}" +
+          "]";
+
+
   /** ApiUnderlay for the underlay specified by the nautical profile. */
   private static final ApiUnderlay NAUTICAL_API_UNDERLAY =
       new ApiUnderlay()
           .name(NauticalUnderlayUtils.NAUTICAL_UNDERLAY_NAME)
           .entityNames(
               ImmutableList.of(
-                  "boat_electric_anchors", "boat_engines", "boats", "reservations", "sailors"));
+                  "boat_electric_anchors", "boat_engines", "boats", "reservations", "sailors"))
+              .criteriaConfigs(criteriaConfigs);
```

Comment:
This looks too far indented to me. It should be on the same level as `.entityNames(` I think. I'm surprised the build process didn't complain.